src/current/_includes/releases/cloud/2024-07-23.md

```diff
@@ -0,0 +1,5 @@
+## July 23, 2024
+
+<h3 id="2024-07-23-general-updates"> General updates </h3>
+
+- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version), which defaults to the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version, including subsequent patches after a Pre-Production Preview, are applied automatically.
```

Comment:
Should we say additional stable "supported" releases? Or something along those lines, or is it implicit that you can only choose releases that we still support.

Reply:
I think it's implicit, you wouldn't be able to select an EOL version even if you have current clusters running that EOL version. And we mention PPP right after, as the other type of release that could be in the list and isn't supported.